test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -1710,18 +1677,26 @@ private RazorViewEngine CreateSuccessfulViewEngine()
 
         private TestableRazorViewEngine CreateViewEngine(
             IRazorPageFactoryProvider pageFactory = null,
-            IEnumerable<IViewLocationExpander> expanders = null)
+            IEnumerable<IViewLocationExpander> expanders = null,
+            IEnumerable<string> viewLocationFormats = null,
+            IEnumerable<string> areaViewLocationFormats = null)
         {
             pageFactory = pageFactory ?? Mock.Of<IRazorPageFactoryProvider>();
             return new TestableRazorViewEngine(
                 pageFactory,
-                GetOptionsAccessor(expanders));
+                GetOptions(expanders, viewLocationFormats, areaViewLocationFormats));
         }
 
-        private static IOptions<RazorViewEngineOptions> GetOptionsAccessor(
-            IEnumerable<IViewLocationExpander> expanders = null)
+        private static IOptions<RazorViewEngineOptions> GetOptions(
+            IEnumerable<IViewLocationExpander> expanders = null,
+            IEnumerable<string> viewLocationFormats = null,
+            IEnumerable<string> areaViewLocationFormats = null)
         {
+            var optionsSetup = new RazorViewEngineOptionsSetup(Mock.Of<IHostingEnvironment>());
```

Comment:
> Suggest instead that this method should add the defaults when viewLocationFormats or areaViewLocationFormats are null

`RazorViewEngineOptionsSetup` is the one which has the default values, so I am using it here. Not sure what is weird about it